README.md

```diff
@@ -21,16 +21,84 @@ Note:
 3. Patches present in features will use prelim uapi to aviod conflict in updates, once patches are merged in drm-tip, uapi will change from prelim to normal.
 4. prelim upai will be maintained at [drm-uapi-helper](https://github.com/intel-gpu/drm-uapi-helper/tree/xe).
 
+# Available Branches
+
+| Kernel Version | Branch Name |
+|----------------|-------------|
+| 6.14 | kernel-backport/main_6.14 |
+| 6.17 | kernel-backport/main |
+
+**Note:** Choose the appropriate branch based on your target kernel version.
+
 # Usage
 backport.sh < options >
 
-|options:| | |
+||options |description |
 |-- |--|--| 
-|1. |create-tree| Create kernel tree based on given option <base/features>|
+|1. |create-tree| Create kernel tree based on given option <base/features> (default)|
 |2. |delete-tree| Delete the tree|
 |3. |reset-tree| Delete the existing tree and re-create it|
 |4. |override| Overrides existing tree|
 
+# Contributing
+
+## Adding Patches
+
+### 1. Create Your Patch
+Generate a patch file from your git commit:
+```bash
+git format-patch -1 <commit-hash>
+```
+
+### 2. Choose Location
+Place the patch in the appropriate directory:
+- **`backport/patches/base/`** - For patches merged in drm-tip
+- **`backport/patches/features/`** - For feature patches under review
+
+### 3. Copy Patch File
+```bash
+# For base patches
+cp <your-patch>.patch backport/patches/base/
+
+# For feature patches
+cp <your-patch>.patch backport/patches/features/
+```
+
+### 4. Update Series File
+Add your patch to the series file at the desired location:
+```bash
+# For base patches
+vim backport/patches/base/series
+
+# For feature patches
+vim backport/patches/features/series
+
+# Add your patch filename in the correct order
+# Example:
+0001-existing-patch.patch
+0002-your-new-patch.patch
```

Comment:
we have only 0001  numbering and series will take care of dependency. So this example should reflect all 0001's